protocols/loader.py

```
@@ -3,28 +3,24 @@
 from utils.logger import log
 
+
 def load_protocol(name):
     """Dynamically load a protocol module by name"""
     try:
         module = importlib.import_module(f"protocols.{name}")
-        return {
-            'name': name,
-            'task': module.task
-        }
+        return {"name": name, "task": module.task}
```

Ruff (0.11.9)

Consider moving this statement to an `else` block

(TRY300)